cis194/week6/zhansongl/Fibonacci.hs

```diff
@@ -14,7 +16,7 @@ fibs1 = map fib [0..]
 
 fibs2 :: [Integer]
 fibs2 = next 0 1
-  where next a b = a : next b (a+b)
+  where next n = (n:) . (<*>) next (n+)
```

Comment:
Composing a infix operator such as `<*>`, `<$>`, `>>=` probably is a sign we are turning the nob too high on composition. But it's a fun exercise.

Reply:
I was trying to figure out if there is a way to remove n as well but was not able to... but yeah, this level of applicative style is probably a bit excessive

Reply:
pointfree.io？

Reply:
oh wow